features/step_definitions/routability.js

```diff
@@ -29,8 +29,13 @@ module.exports = function () {
                                     result[direction].status.toString() : '';
                                 break;
                             case /^\d+s/.test(want):
+                                outputRow[direction] = result[direction].time ?
+                                    result[direction].time.toString()+'s' : '';
                                 break;
                             case /^\d+ km\/h/.test(want):
+                                outputRow[direction] = !isNaN(result[direction].speed) ?
+                                    result[direction].speed.toString()+' km/h' :
+                                    result[direction].speed.toString() || '';
```

Comment:
When can can the speed be NaN here? Why are we appending km/h, I thought the fuzzy matcher was able to handle all of this?